test/Microsoft.Extensions.SecretManager.Tests/ConfigurationExtensionTests.cs

```diff
@@ -18,7 +19,7 @@ public void AddUserSecrets_Does_Not_Fail_On_Non_Existing_File_Explicitly_Passed(
                                 .AddUserSecrets(userSecretsId: Guid.NewGuid().ToString());
         }
 
-        [Fact]
+        [ConditionalFact (Skip = "Obsolete API")]
```

Comment:
You don't need ConditionalFact to do this. `[Fact(Skip = "reason")]`.

Also, don't skip these tests. They should still work even though API is obsolete.